### PR TITLE
Add resend verification

### PR DESCRIPTION
**Pull request by lhurepo**
🎟️ Ticket(s): Closes [#](https://www.notion.so/ubcbiztech/b24732925a6745caa305a2c746fa5ba7?v=d557178d0a134acf85a3890882538364&p=9ee9ac71ebe5422886364f1f4e47239d&pm=s)

👷 Changes: A brief summary of what changes were introduced.
Added a button to resend verification email on the signing page. When an unverified email tries to log in, they will be able to resend it to their email.

💭 Notes: Any additional things to take into consideration.
It is buggin in test mode and after resending, verifying, and logging in, it brings you back to the sign up for membership page

Wait! Before you merge, have you checked the following:

📷 Screenshots
image

(prefer animated gif)

## Checklist

- [ ] Looks good on large screens
- [ ] Looks good on mobile

### CODE REVIEW

File: src/pages/admin/Home.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ function AdminHome(props) {
           <Typography variant="h1" style={{
             color: COLORS.BIZTECH_GREEN
           }}>
-            BizTech Admins
+            BizTech FART
```
Comment (ddennis924) [inline src/pages/admin/Home.js:175]: can u undo this LOL

**Review by ddennis924 — COMMENTED** [review]
resend verification changes look good, but theres a lot of changes here that dont seem relevant to the ticket. Curious as to why they are here? Maybe pull latest dev

File: src/pages/public/RegistrationForms/UserMembershipForm/UserMembershipForm.js [hunk 2]
```diff
@@ -299,7 +293,6 @@ export default function UserMembershipForm(props) {
         <Grid item xs={12}>
           <CustomSelect
             {...props}
-            initialValues=""
```
Comment (ddennis924) [inline src/pages/public/RegistrationForms/UserMembershipForm/UserMembershipForm.js:302]: no sure why these are removed

File: src/pages/admin/Home.js (hunk 1 above)
Comment (lhurepo) [reply src/pages/admin/Home.js:175]: i was wondering what it was cuz i never made changes to home 😂

File: src/pages/public/RegistrationForms/UserMembershipForm/UserMembershipForm.js (hunk 2 above)
Comment (lhurepo) [reply src/pages/public/RegistrationForms/UserMembershipForm/UserMembershipForm.js:302]: will fix

Comment (lhurepo) [reply src/pages/public/RegistrationForms/UserMembershipForm/UserMembershipForm.js:302]: didnt remove them but ok

**Pull request closed without merging**